Editor/main.cpp

```diff
@@ -49,4 +49,4 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
 
 	Window::Destroy();
     return 0;
-}
\ No newline at end of file
+}
```

Comment:
New line added by accident, this file probably doesn't need to be included in the pull request